content/intro-to-storybook/angular/pt/conclusion.md

```diff
@@ -16,12 +16,12 @@ Possui uma comunidade de programadores próspera e uma grande variedade de extra
 
 Pretende saber mais? Aqui ficam alguns recursos que irão ajudar.
 
-- [**Documentação Oficial Storybook**](https://storybook.js.org/basics/introduction/) contém a documentação da API, links da comunidade e a galeria de extras.
+- [**Documentação Oficial Storybook**](https://storybook.js.org/docs/react/get-started/introduction) contém a documentação da API, links da comunidade e a galeria de extras.
```

Comment:
@gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/angular/get-started/introduction" in the "Official Storybook documentation" link.